extensions/interactions/NumericInput/NumericInput.py

```diff
@@ -57,3 +58,285 @@ class NumericInput(base.BaseInteraction):
         'calculation_id': 'Top10AnswerFrequencies',
         'addressed_info_is_supported': True,
     }]
+
+    @classmethod
+    def to_proto(cls, interaction):
+        """Creates a NumericInputInstance proto object.
+
+        Args:
+            interaction: InteractionInstance. The interaction instance
+                associated with this state.
+
+        Returns:
+            NumericInputInstance. The NumericInputInstance proto object.
+        """
+        outcome_proto = interaction.default_outcome.to_proto()
+
+        hints_proto_list = []
+        for hint in interaction.hints:
+            hint_proto = hint.to_proto()
+            hints_proto_list.append(hint_proto)
```

Comment:
ditto as above, also maybe some of these similar chunks of code can be introduced into `BaseInteraction`?

Reply:
Done